### PR TITLE
Updated support for Tioga Pass

### PR DESCRIPTION
**Pull request by johnnylinwiwynn**
The initrd is embedded in the bzImage in this method.

### CODE REVIEW

**Comment by johnnylinwiwynn** [comment]
@jonzhang-fb @insomniacslk @dhendrix @morganjangwiwynn

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ BOARD		?= qemu
 KERNEL		?= bzImage
 INITRD		?= initrd.cpio.xz
 BUILD		:= build/$(BOARD)
-
+EXFW_OPT	?=
```
Comment (zaolin) [inline Makefile]: Unrelated changes, please split it in two patches

File: Makefile.uefi [hunk 2]
```diff
@@ -16,7 +16,7 @@ EfiSmmAccess2ProtocolGuid	:= c2702b74-800c-4131-8746-8fb5b89ce4ac
 EfiSmmControl2ProtocolGuid	:= 843dc720-ab1e-42cb-9357-8a0078f3561b
 IntelRegAccessProtocolGuid	:= FD480A76-B134-4EF7-ADFE-B0E054639807
 EfiMpServiceProtocolGuid	:= 3fdda605-a76e-4f46-ad29-12f4531b3d08
-IntelPlatformTypeGuid		:= a7ced760-c71c-4e1a-acb1-89604d5216cb
+IntelPlatformTypeGuid		:= 171e9398-269c-4081-9099-3844e260466c
```
Comment (zaolin) [inline Makefile.uefi]: Why?

File: Makefile [hunk 3]
```diff
@@ -115,6 +115,7 @@ $(BUILD)/$(BOARD).txt: $(ROM)
 	( \
 	cd $(BUILD) ; \
 	$(pwd)/bin/extract-firmware \
+		$(EXFW_OPT) \
```
Comment (zaolin) [inline Makefile]: Same here

File: Makefile.uefi (hunk 2 above)
Comment (johnnylinwiwynn) [reply Makefile.uefi]: IntelPlatformTypeGuid is the same as EfiIioUdsProtocolGuid, so we checked the UEFI BIOS code base and found IntelPlatformTypeGuid should be 171e9398-269c-4081-9099-3844e260466c.
@morganjangwiwynn

File: Makefile (hunk 3 above)
Comment (johnnylinwiwynn) [reply Makefile]: Here we intend to add "--repack" parameter for extract-firmware, because the FVs of AMI BIOS are compressed by LZMA format, but LZMA decompress protocol is not built by default in the EDK2 code, so adding "--repack" parameter to decompress the FVs, otherwise the Linuxboot BIOS can`t boot up on TiogaPass.
@morganjangwiwynn

File: Makefile (hunk 1 above)
Comment (johnnylinwiwynn) [reply Makefile]: It's for adding "--repack" for , here we set it to blank by default, I can moved it to another patch.

**Comment by johnnylinwiwynn** [comment]
Need to split and re-base this change on another new change, so close this PR.
